### DISCUSSION TITLE
Bliss 2 font open source?

### DISCUSSION THREAD

**Discussion by elegeria** [community/community#119913] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Are these font [https://github.com/sayedwasim/NCA/tree/master/css/fonts](url) open source?

**Comment by ebndev** [comment]
Hi @elegeria, I don't see an open-source license on that repository but it looks like they are using the "Bliss" font family and "Icomoon" icon library, which can be be downloaded for free if you search for them on Google.

**Reply by elegeria** [reply]
Thank u!
